apps/annotated_retry/README.md

```diff
@@ -0,0 +1,31 @@
+# AnnotatedRetry
```

Comment:
SMDH thought they were just using the actual OG retry

Reply:
So they were able to merge these changes into original retry, but they did not integrate that into hindsight at this point so that is something I can refactor in but i didnt bite that off yet.